commands/pull_request.go

```diff
@@ -208,7 +212,13 @@ func pullRequest(cmd *Command, args *Args) {
 		}
 
 		utils.Check(err)
+
 		pullRequestURL = pr.HTMLURL
+
+		if flagPullRequestAssignee != "" {
+			err = client.UpdateIssue(baseProject, pr.Number, flagPullRequestAssignee)
```

Comment:
Right now you don't have to worry about other fields. We just need one field, so make a specialized method, and it will be easy for future generations to expand on this approach to add other features.